Codemine/Extensions/CGRect+Utilities.swift

```diff
@@ -35,7 +35,7 @@ public extension CGRect {
      
      - returns: a new CGRect with the width and height reversed to those of the current one
      */
-    public func rectByReversingSize() -> CGRect {
+	public var reversedSize: CGRect {
```

Comment:
This sounds a bit odd, as if it was return a size instead of `CGRect`. Is `reverseSized` grammatically incorrect and/or too weird? :D